boilerplate/bin/downloadExpoApp.sh

```diff
@@ -1,18 +1,20 @@
 #!/bin/bash -e
 
-# taken from https://github.com/expo/with-detox-tests/blob/master/setup.sh
+# based on https://github.com/yaron1m/expo-detox-typescript-example/blob/master/scripts/dl_expo_bins
+APP_PATH=./bin/Exponent.app
```

Comment:
Do we know that they'll be running this from the root folder? If not, we may want to add a check for that (see if `package.json` exists, for example).

Reply:
Good idea! Will do

Reply:
TIL: Both `yarn` and `npm` execute this script in the project root even if the user runs `yarn test:e2e` or `npm run test:e2e` from a project subdirectory (but I'm still adding the check!).

Reply:
✅